util/constants/quality-id-to-measure-map.js

```diff
@@ -0,0 +1,18 @@
+// Libraries
+var keyBy = require('lodash/keyBy');
+// Data
+var measures = require('../../measures/measures-data.json');
+
+/**
+ * @type {{}} - mapping of integer qualityIds to corresponding measure
+ */
+var QUALITY_ID_TO_MEASURE_MAP = keyBy(measures, function(measure) {
```

Comment:
Rather than put this in a separate file can you put it in `format-csv-records`? Since it is only used there.